assets/training/finetune_acft_image/components/pipeline_components/medimage_insight_ft/spec.yaml

```diff
@@ -0,0 +1,141 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
+name: medimage_insight_ft_pipeline
+version: 0.0.1
+type: pipeline
+display_name: Medimage Insight Finetune Pipeline
+description: Pipeline Component to finetune MedImageInsight Model.
+
+inputs:
+  mlflow_embedding_model_path:
+    type: uri_folder
+    optional: false
+    description: Path to the MLflow model to be imported.
+    mode: ro_mount
+
+  eval_image_tsv:
+    type: uri_file
+    optional: false
+    description: Path to the evaluation image TSV file.
+    mode: ro_mount
+
+  eval_text_tsv:
+    type: uri_file
+    optional: false
+    description: Path to the evaluation text TSV file.
+    mode: ro_mount
+
+  image_tsv:
+    type: uri_file
+    optional: false
+    description: Path to the image TSV file.
+    mode: ro_mount
+
+  text_tsv:
+    type: uri_file
+    optional: false
+    description: Path to the text TSV file.
+    mode: ro_mount
+
+  label_file:
+    type: uri_file
+    optional: false
+    description: Path to the label file.
+    mode: ro_mount
+
+  conf_files:
+    type: uri_file
+    optional: false
+    description: Path to the configuration files.
+    mode: ro_mount
+
+  # preprocessing settings
+  image_standardization_jpeg_compression_ratio:
+    type: integer
+    optional: true
+    default: 75
+    description: JPEG compression ratio for image standardization.
+
+  image_standardization_image_size:
+    type: integer
+    optional: true
+    default: 512
+    description: Image size for standardization.
+
+  instance_type_finetune:
+    type: string
+    optional: true
+    default: Standard_nc24rs_v3
+    description: Instance type to be used for finetune component in case of serverless compute, eg. standard_nc24rs_v3.
+      The parameter compute_finetune must be set to 'serverless' for instance_type to be used
+
+  compute_finetune:
+    type: string
+    optional: true
+    default: serverless
+    description: compute to be used for finetune eg. provide 'FT-Cluster' if your
+      compute is named 'FT-Cluster'. Special characters like \ and ' are invalid in the parameter value.
+      If compute cluster name is provided, instance_type field will be ignored and the respective cluster will be used
+
+  process_count_per_instance:
+    type: integer
+    min: 1
+    default: 1
+    optional: true
+    description: Number of processes to run per instance. This is used to set the number of GPUs to use for training.
+
+  instance_count:
+    type: integer
+    min: 1
+    default: 1
+    optional: true
+    description: Number of instances to use for training.
+outputs:
+  save_dir:
```

Comment:
how is this different from below two? Could you add more detailed description how they gets used?